cabal-install/Main.hs

```diff
@@ -1265,5 +1265,4 @@ manpageAction :: [CommandSpec action] -> Flag Verbosity -> [String] -> Action
 manpageAction commands _ extraArgs _ = do
   unless (null extraArgs) $
     die $ "'manpage' doesn't take any extra arguments: " ++ unwords extraArgs
-  pname <- getProgName
-  putStrLn (manpage pname commands)
+  putStrLn (manpage "cabal" commands)
```

Comment:
Maybe use `takeBaseName` instead? Use case: a user has both `cabal-1.18` and `cabal-1.20` installed.